.github/workflows/ci_cd_release.yml

```
@@ -221,8 +243,10 @@ jobs:
     name: "Deploy stable documentation"
     runs-on: ubuntu-latest
     needs: release
+    permissions:
+      contents: write
     steps:
-      - uses: ansys/actions/doc-deploy-stable@main
+      - uses: ansys/actions/doc-deploy-stable@feat/check-actions-security-action
```

```suggestion
      - uses: ansys/actions/doc-deploy-stable@main
```
Needs to be reverted before merging